packages/dev-server/package.json

```diff
@@ -33,34 +33,28 @@
     "@babel/code-frame": "^7.16.7",
     "@fastify/middie": "^8.3.0",
     "@fastify/sensible": "^5.5.0",
-    "@react-native/dev-middleware": "^0.73.8",
+    "@react-native/dev-middleware": "^0.76.1",
     "fastify": "^4.24.3",
     "fastify-favicon": "^4.3.0",
     "fastify-plugin": "^4.5.1",
-    "open": "^8.4.0",
     "pretty-format": "^28.1.0",
     "source-map": "^0.7.4",
     "ws": "^8.7.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.23.9",
     "@babel/core": "^7.23.9",
-    "@react-native-community/cli-debugger-ui": ">=11.3.1",
-    "@react-native-community/cli-server-api": ">=11.3.1",
+    "@react-native-community/cli-server-api": "15.0.1",
     "@types/babel__code-frame": "^7.0.3",
     "@types/node": "18",
     "@types/ws": "^8.5.3",
     "babel-plugin-add-import-extension": "^1.6.0",
     "typescript": "^5.5.3"
   },
   "peerDependencies": {
-    "@react-native-community/cli-debugger-ui": ">=11.3.1",
     "@react-native-community/cli-server-api": ">=11.3.1"
```

Comment:
bump to 15.0.1

Reply:
do we want to bump it tho? We could bump to version that was associated with RN 0.74 🤔

Reply:
bumped to 13.6.4 which is associated with 0.74 👍